Makefile

```diff
@@ -305,13 +315,21 @@ DEF_INC_CFLAGS := $(foreach i,$(INCLUDE_DIRS),-I$(i)) $(C_DEFINES)
 ifneq (,$(call find-command,clang))
   CPP      := clang
   CPPFLAGS := -E -P -x c -Wno-trigraphs $(DEF_INC_CFLAGS)
-else
+else ifneq (,$(call find-command,cpp))
```

Comment:
Can you explain this change? Why does it do?

Reply:
just to say when the user don't have clang or cpp